tutor/src/screens/assignment-review/homework-scores.js

```diff
@@ -168,9 +168,7 @@ const StudentCell = observer(({ ux, student, striped }) => (
       </NameWrapper>
 
       <Total>
-        {ux.displayTotalInPercent ?
-          `${S.asPercent(student.total_fraction || 0)}%` :
```

Comment:
I think my original fallback here to `0` was incorrect, and it should always defer to if BE sent the value or not (show the value if we have it, otherwise `---`)